blivet/osinstall.py

```diff
@@ -250,7 +250,7 @@ def _find_existing_installations(devicetree):
 
         if not os.access(get_sysroot() + "/etc/fstab", os.R_OK):
             util.umount(mountpoint=get_sysroot())
-            device.teardown(recursive=True)
+            device.teardown()
```

Comment:
In `find_existing_installations` there is a `teardown_all` in a `finally` block around the call to `_find_existing_installations`, so everything should get deactivated regardless of outcome.